### DISCUSSION TITLE
Equipamento para Dev Nômade

### DISCUSSION THREAD

**Discussion by diegoemerique** [frontendbr/forum#237] [Discussão]
Fala galera! Sei que a pergunta talvez n se encaixe aqui, mas gostaria de uma dica dos manos e minas que curtem uns remoto e sair por ai codando ;)

Qual seria um notebook(ou uma configuração) adequada para WebDevs, seja front ou back, que suprisse bem os quesitos:
- Tempo fora da tomada ( bateria )
- Performance
- Preço ( já que poderia ser roubado )

Fico pensando pq  já que o pais está muito perigoso, andar por ai portando um MacBook Pro está cada dia mais difícil, mas por outro lado, um note muito barato n aguentaria o trampo de um DEV de Starbucks(haha), tendo em vista q ele vai ficar horas fora da tomada. (Deixando claro que não é só para parecer Cool, as vezes algumas pessoas precisam desse tipo de coisa, pois n param em casa. E nos locais que pode parar n tem tomada a disposição).

Qual seria a opinião de vcs!?

**Comment by barbier** [comment]
Procure um com a sexta geração da Intel, eles fizeram magia negra com o consumo de bateria.
A Dell já começou a fazer notebooks com bateria equivalente à dos Macbooks, vale a pena dar uma olhada....
Edit: Eu diria que ter 8Gb de RAM é essencial hoje em dia, mas não sei como é a diferença de desempenho "Mais Ram e HD" x "Menos RAM e SSD".

Se for carregar o notebook 24/7, ou perto disso, faça seguro. É barato, evita dor de cabeça e lembre-se sempre de manter o seu código atualizado na nuvem!
(dependendo do tipo de projeto que você use, colocar uma criptografia de boot é uma boa também....)

**Comment by diegoemerique** [comment]
Gostei dos da DELL e parece estar em um preço bom também.

Sobre o seguro, eu não tinha pensado sobre isso, para falar a verdade eu nem sabia que tinha como. Achei bem interessante!

**Comment by woliveiras** [comment]
Rapaz, já tive a experiência de `RAM++` e `SSD` no trampo e foi muito bom!

Vale muito a pena investir em um PC com bastante RAM e SSD, mesmo que seja um SSD de 120, por exemplo, pois você pode manter seus arquivos na nuvem, caso precise de mais armazenamento, e vai ter uma performance muito melhor.

Quando tive a experiência de trabalhar com mais memória e SSD trabalhava com Windows e até aquelas engasgadas do S.O. na cópia de arquivos muito grandes do Pen Drive para o Disco pararam!!!

`Quanto ao preço:` O Note é uma ferramenta de trabalho, imagina trabalhar com uma ferramenta ruim por medo de ser assaltado... Não é legal.

Acho que seria interessante pensar num seguro pra sua máquina, criptografar até papel de bala que você comer perto do note, manter coisas na nuvem pra não perder e pegar um PC bom.
Da uma olhada nessa opção da [Porto Seguro](http://www.portoseguro.com.br/seguros/equipamentos-portateis/sobre-o-produto/seguro-para-notebook). Não sei se é a melhor opção, mas vejo boas indicações da Porto.

`Marca:` Eu iria de DELL por ter boa compatibilidade com as distros Linux que já usei. Acer também está com alguns PCs fodelões por um bom preço, mas nunca usei PCs da marca, então não sei se seria a mesma qualidade. Também não sei como anda a qualidade dos PCs da DELL, né? Vale a pena aguardar o comentário do pessoal quanto a marcas legais.

**Comment by felipefialho** [comment]
@woliveiras Tenho um DELL, com Windows 10 :confused: 

Mas com relação a performance, é muito boa. Estabilidade também.

As configurações:
- Monitor 15,6' com Touch Display, IPS
- Resolução Full HD (1920 x 1080)
- Processador Intel Core i7
- 16GB de memória
- Placa de vídeo AMD Radeon M265 2GB DDR3
- 1 TB de HD + 8GB SSD 

O maior problema é o peso do Notebook, o @willianjusten pode confirmar :smile:

**Comment by kelciocajueiro** [comment]
Ja tive um Acer em meados de 2010. Com pouco mais de 2 anos de uso ele começou a apresentar problemas: a bateria não durava nem 10 minutos, esquentava a ponto de desligar o notebook e por fim o HD do nada não iniciava mais o SO. Tentei abrir ele e acabei quebrando todo. Sem falar que ano passado eu comprei um tablet da Acer para minha esposa utilizar com uns clientes dela e com 1 semana tive que mandar pra autorizada. Resumindo, perdi minha total credibilidade com a marca Acer.

Em 2013 comprei um ASUS K45VM (core i7, 8gb RAM, 1TB HD, etc, etc...) e até hoje ele está firme e forte. Não pretendo trocar de equipamento por enquanto. Mas quando for trocar vou dar uma olhada com carinho nos produtos da ASUS.

Ja vi muitas pessoas falando bem da Avell, dei uma olhada rápida e todos os aparelhos dessa marca são de alto desempenho. Dá uma conferida la :)

**Comment by woliveiras** [comment]
Essa configuração está monstruosa @LFeh!

Os Avell são bem potentes @kelciocajueiro. A maioria é pra jogo, ai já viu... Só monstro.

Putz, então já tiramos da lista os Acer's. kkkk

Eu comprei um Asus pra minha mãe e tem durado muito, apesar do pouco uso dela, quando a marca é ruim da problema independente do uso né? 😄 

O meu mesmo eu nem comento, é quase mais véio que minha mãe e bem fraquinho. UHAHUUAHA

**Comment by barbier** [comment]
Eu não recomendo Avell para trabalho, a menos que você vá trabalhar com jogos, porque eles são mais caros, você não vai usar a placa de vídeo e chuto que a bateria não vá durar muito tempo.... Eu tive um LG com um i7 (quad-core com HT, na prática eram 8 núcleos), 6gb de RAM e uma placa da AMD cujo modelo não lembro. A bateria não durava NADA, 1h chorando e pedindo com muita gentileza....

Foi bom vocês tocarem no assunto "peso", eu tinha esquecido disso.... Tente evitar notes com mais de 2kg, sua coluna agradece! Os processadores Intel que tenham U no modelo, não lembro se prefixo ou sufixo, são mais econômicos e não perdem muito desempenho em relação ao padrão, pode ir nesses.

**Comment by mandado** [comment]
@LFeh  tamo junto! rs , uso também esse note. muito bom por sinal. só to tendo um problema com o touch pad no linux, que ele rola com um dedo do nada ao invés de dois dedos, ai tenho que desligar e ligar o touchpad rs. ai por isso acabo usando mouse, imagino que no windows não tenha esse problema.

**Comment by felipefialho** [comment]
@barbier É que não tenho muito costume de trabalhar com ele no colo ou me locomover muito. Mas se tivesse, com certeza seria uma merda.

**Comment by barbier** [comment]
@LFeh acredito!
Esse meu note eu usava como desktop, apesar da tela ser de 14". Eu o coloquei numa base ao lado do meu monitor principal, de 27", alinhei fisicamente e digitalmente porque tenho toc com isso (:'D) e só usava na tomada... Apesar da bateria inexistente, é uma máquina boa, tanto que a minha irmã usa até hoje nos projetos dela.

**Comment by rubenmarcus** [comment]
esse seguro vale pra quando voce sair do pais tambem? 
achar internet boa não é o melhor problema?

**Comment by woliveiras** [comment]
Tem a opção de cobertura Internacional @rubenmarcus, da uma olhada nessa cobertura: http://www.portoseguro.com.br/seguros/equipamentos-portateis/saiba-mais/coberturas

Mas isso pela Porto, da vale a pena dar uma pesquisada maior e ver quais as opções viu.

**Comment by luisfmsouza** [comment]
Faz sentido incluir nessa discussão bons lugares pra se trabalhar? Citando:
- Qualidade da internet
- Preço
- Localização

Ex.: Starbucks com boa internet em Campinas: https://goo.gl/maps/ddZhVudJUUR2.

**Comment by felipefialho** [comment]
@luisfmsouza Gostei da ideia, não quer abrir uma issue e referenciar essa?

**Comment by luisfmsouza** [comment]
Criei a #241 @LFeh. Edita lá se precisar :)

**Comment by willianjusten** [comment]
Melhor Notebook para trabalhar remoto (_na minha opinião_) é e ainda será durante um bom tempo o **Macbook Air**.
- 12 horas de bateria de verdade (um monte diz que dura e num chega nem em 6h)
- WiFi 802.11ac que é mais rápido e tem maior alcance
- Já vem SSD por padrão
- Tem o melhor trackpad em comparação a qualquer um, você vai trampar remoto, então num vai levar mouse para lá e para cá, pense na raiva que você pode passar com um trackpad ruim.
- Acabamento perfeito de alumínio e unibody, sem as chances de quebrar peças ou soltar com eventuais batidas na mochila.
- Pesa só 1,35 kg e menos de 2cm de largura! (Ninguém vê que vc tem um notebook na mochila e facilita para locomover)

**Comment by Adejair** [comment]
@willianjusten Eu tenho um Macbook Air, a bateria dura em torno de 5 horas / 5:30 com o brilho no máximo, com o brilho mais reduzido, por volta da metade, chega a durar 6 até 6:30.

Para min o aparelho é o melhor, pelo tamanho e pelo peso, você pode comprar a espessura dele a uma caneta BIC(Aôôô BIC, paga nois pelo merchan uasha), então para se locomover fica muito mais fácil, desempenho nem se fala.

**Comment by thulioph** [comment]
Reforçando o que já foi dito por @willianjusten e @Adejair, o Air é sem dúvida um excelente investimento. 

Quem precisa andar com sua máquina e principamente anda de coletivo, vale a pena considerar: peso, espessura e tamanho; Já como ferramente de trabalho eu gosto de observar também: duração da bateria, processador e se o teclado é retroiluminado (ajuda muito pra quem trabalha a noite).

Uma observação sobre o seguro: se você compra no site da apple, tem o [Apple Care](http://www.apple.com/br/shop/browse/home/applecare) que é uma alternativa.

**Algumas considerações:**

✔️ Eu não gosto de mochilas que chamam muita atenção, dependendo de onde você mora, ou o percurso que você faz, é até mais interessante optar por uma mochila "mais comum".

✔️ Na mochila, além da sua máquina tente levar um livro ou um casaco dentro também, dá mais volume e ajuda a desfarçar. (também pode ser útil no tédio/frio rsrs) 

✔️ Para ajudar na durabilidade da máquina, você pode optar por comprar um [case para sua máquina](http://lista.mercadolivre.com.br/cases-para-macbook#D[A:cases-para-macbook]), assim pode colar adesivo a vontade, quando tirar a capa, a máquina ainda estará nova E caso ele caia ou algo bata em cima, ajuda a proteger E se você quiser vender um dia, estará bem conservada.

✔️ A bateria é influenciada muito pela quantidade de processos ativos, usando Spotify, com o Mongodb, Grunt, Skype, Mail, TweetDeck, etc.. não tem bateria que aguente.

No mais, tome bastante cuidado com sua máquina pois é um verdadeiro investimento (se vê pelo preço). 😄

**Comment by rubenmarcus** [comment]
esse macbook air vi o mais barato por 5k no Mercado Livre..

esse aqui pra mim parece ser um bom custo beneficio...

http://www.kalunga.com.br/prod/notebook-yoga-500-processador-core-i5-(5%20-geracao)-de-22ghz-memoria-de-8gb-hd-de-1tb-tela-de-14-touch-lenovo/221464

http://www.americanas.com.br/produto/124784337/notebook-samsung-expert-x22-intel-core-i5-8gb-1tb-led-hd-15-6-windows-10-branco?nm_ranking_rec=2&nm_origem=rec_item_page.rr1-CategorySiloedViewCP&DCSext.recom=RR_item_page.rr1-CategorySiloedViewCP

esse tambem. Eu como tambem edito fotos e videos como hobby, ter um laptop com espaco em hd, e boa memoria e resolucao 1920x1080 e o minimo. Vi uns macbook air com 1366 de resolucao, e ridiculo vc vai trampar com design responsivo e um saco

Essa linha Yoga e esses notes que viram tablet, sao recomendaveis? 
Alem de trampar com web, eu tb faco umas ilustras , tiro uns decalques pra tatuagem direto da tela,quando nao imprimo o desenho, pra mim ia ser uma mao na roda!

**Comment by tgmarinho** [comment]
Olá, para manter a thread, e ai em 2018 o que rola? 

Mac air, pro? dell? lenovo, thinkpad?

Até 5k (novo/usado) qual a opinião de vocês?

**Comment by renanlara** [comment]
@woliveiras, não vamos já de cara remover a Acer, lembre-se que o @kelciocajueiro comentou que o que ele disse foi em meados de 2010, estamos em 2018 já :)

Eu tenho um Acer já faz 2 anos, está puramente conforme veio de fábrica, só andei formatando pois estava fazendo uns testes com Linux. Configs:

- Intel i7
- 8GB de RAM
- 1TB de HD
- GeForce 920M com 2GB de VRAM Dedicada
- Placa de Vídeo Intel integrada

Eu trabalho alocado com ele na empresa mas levo e trago para casa todos os dias. Ele tem aguentado essa pancadaria já o tempo todo e nunca me deu dor de cabeça.

**Comment by giovannism20** [comment]
@renanlara Não pode esquecer da questão da bateria. Em quesito configuração X performance, parece bom mas e quanto a bateria, duração. Não esqueça que o mesmo estará na Starbucks; (@starbucks patrocina aqui.)

**Comment by willycamargo** [comment]
Não me considero fanboy da Apple, inclusive fui muito resistente por um tempo sobre utilizar produtos tão caros. Achava desnecessário. Mas, depois dos primeiros contatos, logo percebi que o valor de investimento vale a pena.

O tempo despendido configurando ambiente + corrigindo bugs + procurando softwares equivalentes + etc, fazem com que ambientes sem Mac OS se tornem absurdamente menos produtivo a longo prazo.

Trabelhei por meses com Windows, por anos com Ubuntu e agora estou há 2 anos com mac.

Minha lógica seria a seguinte: sempre procurar um macbook usado com as melhores configurações possíveis dentro do meu orçamento

- Se eu tiver 5k, provavelmente macbook pro de 13 Mid 2015, com i7, 16gb e 256gb SSD
- Se eu tiver 3k, provavelmente um macbook air de 13 Mid 2015, com i5, 8gb de RAM e 256gb SSD
- Se eu tiver 2k, provavelmente um pro de 13 de 2010, com i3, 8gb de RAM e 500gb HD.

Enfim, procuraria o melhor macbook dentro do orçamento.
Meu único arrependimento foi ter comprado um macbook pro de 15" (que inclusive estou vendendo). O computador é muito grande e pesado e estou sempre saindo com ele para eventos, reuniões, coworking etc. Um macbook de 13 atenderia minhas expectativas sem dificuldades.

**Comment by tgmarinho** [comment]
Legal @willycamargo díficil achar mac Mid 2015 com 16 GB de RAm e 256 SSD por 5 mil reais, se eu achasse teria comprado, infelizmente o pro retina 2015 não aumenta a memória pois está embutida na placa mãe, senão eu teria pego um de 8 GB e feito upgrade. Acabei indo de Dell 14 7000 core i7 8ª, 16 GB RAM, 128 SSD, R$ 4.000, mas talvez eu venda mais pra frente pra comprar um mac com essa configuração que vc citou acima, acho que limita 8 GB hoje em dia, no Ubuntu e Deepin já tenho sentido falta, devido ao Chrome goluso hehe

@renanlara cara coloca um SSD ai vai ficar top, por 350 vc compra um de 256 GB
